OFL.txt

```diff
@@ -1,9 +1,9 @@
-Copyright (c) 2022, GitHub https://github.com/github/mona-sans
-with Reserved Font Name "Mona Sans"
+Copyright 2022 The Mona Sans Project Authors (https://github.com/github/mona-sans), with Reserved Font Name "Mona Sans"
```

Comment:
Please remove ` Sans`, as any derivative couldn't use it without permission which seems awkward. 

(Needs to updated in the .glyphs source file also)